packages/connect-extension-protocol/tsconfig-mjs.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "esnext",
+    "outDir": "./dist/mjs",
+    "target": "esnext"
```

```suggestion
    "target": "es2018"
```

Let's keep the values that were in `tsconfig.json`